test/test_jit.py

```diff
@@ -2242,7 +2242,7 @@ def fn(x):
                 warnings.warn("x is less than 2")
             return x
 
-        self.assertExpectedGraph(fn.graph)
+        self.assertTrue(str(fn.graph).count("aten::warn") == 1)
```

Comment:
can we also make this a File check count? it would be nice if we stick one test syntax